### PR TITLE
update readme and tokens for integration test

### PR DESCRIPTION
**Pull request by bhtibrewal**
Date: 6April

Developer Name: @bhtibrewal 

---

## Issue Ticket Number
https://github.com/Real-Dev-Squad/skill-tree-backend/issues/112

### Documentation Updated?

- [ ] Yes
- [ ] No

### Under Feature Flag

- [ ] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [ ] No

### Breaking Changes

- [ ] Yes
- [ ] No

### Development Tested?

- [ ] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

File: README.md
Comment (satyam73) [inline README.md:163]:
```suggestion
  > Note : Publickey in both [RDS backend](https://github.com/Real-Dev-Squad/website-backend) and skilltree backend should be the same.
```

File: README.md
Comment (satyam73) [inline README.md:176]:
```suggestion
Using [Website - backend](https://github.com/Real-Dev-Squad/website-backend)
```

File: README.md
Comment (satyam73) [inline README.md:178]:
```suggestion
This will create a cookie in your browser named `rds-session-v2-development`
```

File: README.md
Comment (satyam73) [inline README.md:181]:
```suggestion
This will create a cookie in your browser named `rds-session-v2-staging`
```

File: README.md
Comment (satyam73) [inline README.md:180]: Please change status to our skilltree staging site

**Comment by satyam73** [comment]
Please fill the readme

**Review by bhtibrewal — COMMENTED** [review]
update auth link

File: README.md
Comment (Chethan64) [inline README.md:180]: Any reason why you're changing this back to `MYSQL_USERNAME`? I had changed it to `MYSQL_ROOT_USERNAME` in my previous PR since setup was failing.

File: README.md
Comment (akhilkh2000) [inline README.md:201]: Minor but shouldn't this be http and not https?
```suggestion
On Local - http://localhost:3000/auth/github/login?redirectURL=https://staging-skilltree.realdevsquad.com/tasks?v2=true
```

Comment (akhilkh2000) [inline README.md:201]: Just wanted to understand shouldn't the redirect URL on local also be `localhost:3000/tasks?v2=true`

**Review by akhilkh2000 — COMMENTED** [review]
@bhtibrewal  - Could we also squash the commits before merging? This will help our commit history cleaner going forward.

File: README.md
Comment (bhtibrewal) [reply README.md]: no redirect URL need not be of the localhost only

Comment (akhilkh2000) [reply README.md]: Yeah it would still work, but to keep consistency with the env, would we not want it to be localhost itself? Can you help me understand if there is any reason to keep it staging? Just wondering as it confused me.  If it will work with localhost, can we change it for consistency? Shouldnt use our staging env for the redirect if we are on local right? I

Comment (akhilkh2000) [reply README.md]: Discussed offline. TLDR : This is just a frontend redirection URL (could be anything else as well so no concerns there) and V2 param is used by the website backend to set the cookie.

**Review by akhilkh2000 — APPROVED** [review]
Approved, could you please squash once before merging though? It'll help keep our commit history clean going forward.

**Pull request merged**
